packages/fxa-shared/subscriptions/types.ts

```diff
@@ -14,7 +19,26 @@ export interface RawMetadata {
 }
 
 // A mapping of OAuth client ids to their corresponding capabilities.
-export type ClientIdCapabilityMap = Record<string, string[]>;
+export type ClientIdCapabilityMap = Record<string, readonly string[]>;
+export const ClientIdCapabilityMap = {
```

Comment:
Judging by the name of the file, I _think_ it's not meant to have function implementations?  Not sure though.